File: app/src/main/java/com/bevstudio/wolfbooksapp/model/api/Offer.java
package com.bevstudio.wolfbooksapp.model.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Offer {

    @SerializedName("finskyOfferType")
    @Expose
    private Integer finskyOfferType;
    @SerializedName("listPrice")
    @Expose
    private ListPrice__1 listPrice;

    public Integer getFinskyOfferType() {
        return finskyOfferType;
    }

    public void setFinskyOfferType(Integer finskyOfferType) {
        this.finskyOfferType = finskyOfferType;
    }

    public ListPrice__1 getListPrice() {
        return listPrice;
    }

    public void setListPrice(ListPrice__1 listPrice) {
        this.listPrice = listPrice;
    }

}

File: app/src/main/java/com/bevstudio/wolfbooksapp/model/api/Offer.kt
package com.bevstudio.wolfbooksapp.model.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Offer {
    @SerializedName("finskyOfferType")
    @Expose
    var finskyOfferType: Int? = null

    @SerializedName("listPrice")
    @Expose
    var listPrice: ListPrice__1? = null
}